engine/execution/execution_test.go

```diff
@@ -105,10 +105,10 @@ func TestExecutionFlow(t *testing.T) {
 	// check collection node received the collection request from execution node
 	providerEngine := new(mocknetwork.Engine)
 	provConduit, _ := collectionNode.Net.Register(engine.ProvideCollections, providerEngine)
-	providerEngine.On("Submit", exeID.NodeID, mock.Anything).
+	providerEngine.On("Submit", mock.Anything, exeID.NodeID, mock.Anything).
```

Comment:
In all places we mock for channel name, instead of `mock.Anything` we should either use `mock.AnythingOfType("network.ChannelName")` or do a type assertion for `args.Get(0).(network.ChannelName)`. 

In general, it is very unsafe to leave a certain parameter at `mock.Anything` all over the tests.

Reply:
Thanks, resolved.